pkgs/by-name/me/meow/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  config,
+  pkgs,
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  ...
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "meow";
+  version = "v2.1.3";
+  meow_owner = "PixelSergey";
+  meow_repo = "meow";
+
+  src = fetchFromGitHub {
+    owner = meow_owner;
+    repo = meow_repo;
+    rev = version;
+    hash = "sha256-PB871c137uxxPaYbV6NB8kECVUrQWTeVz0ciVLHrph4=";
+  };
+
+  cargoHash = "sha256-Yjpjkk5qEX7wU/0M+9PEyzwtQIPQhSng1RFx33KFArc=";
+
+  cargoPatches = [
+    ./Cargo.lock.patch
+    ./Cargo.toml.patch
+  ];
+
+  meta = {
+    description = "Print ASCII cats to your terminal ";
+    homepage = "https://github.com/PixelSergey/meow";
+    license = lib.licenses.mit;
+    maintainers = [ "PixelSergey <sergey.ichtchenko@gmail.com>" ];
```

Comment:
You should add yourself to the list of maintainers here: [maintainer-list.nix](https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix). After that, include your name in the `meta.maintainers` list like this:

```nix
maintainers = with lib.maintainers; [ pixelsergey ];
```

Reply:
Fixed!